packages/cli/src/lib/implementation/config-middleware.spec.ts

```diff
@@ -1,33 +1,34 @@
-import { join } from 'path';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
 import { expect } from 'vitest';
 import { configMiddleware } from './config-middleware';
-import { getDirname } from './helper.mock';
-
-const __dirname = getDirname(import.meta.url);
 
+const __dirname = dirname(fileURLToPath(import.meta.url));
 const withDirName = (path: string) => join(__dirname, path);
 const config = (ext: string) =>
-  `${withDirName('../../../test/config.mock.')}${ext}`;
+  `${withDirName(
```

Comment:
This path notation will break on Windows. Depending which PR gets merged first, I can fix it in mine when I rebase.

Reply:
quick fixed... please consider it in your PR